cmd/createtree/main.go

```diff
@@ -141,16 +142,28 @@ func newPK(opts *createOpts) (*any.Any, error) {
 	switch opts.privateKeyType {
 	case "PEMKeyFile":
 		if opts.pemKeyPath == "" {
-			return nil, errors.New("empty PEM path")
-		}
-		if opts.pemKeyPass == "" {
-			return nil, fmt.Errorf("empty password for PEM key file %q", opts.pemKeyPath)
+			return nil, errors.New("empty pem_key_path")
 		}
 		pemKey := &keyspb.PEMKeyFile{
 			Path:     opts.pemKeyPath,
 			Password: opts.pemKeyPass,
 		}
 		return ptypes.MarshalAny(pemKey)
+	case "PrivateKey":
```

Comment:
Would mind making this the default (line 61)? @RJPercival and I discussed this a bit, and we both seem to prefer it slightly over PEMKeyFile. (I'm guessing you do too.)